src/cromshell/status/command.py

```diff
@@ -114,34 +115,74 @@ def main(config, workflow_id):
     return ret_val
 
 
-def get_metadata_status_summary(workflow_metadata):
+def get_all_call_values(nested_dictionary):
+    """Summarizes a given workflow metadata dictionary into a
+    new dictionary with only task name and status"""
+    temp_dic1 = {}
+    temp_dic2 = {}
+    temp_dic3 = {}
+    for key, value in nested_dictionary.items():
+        # Skip any keys that have values that are strings, because we are note
+        # interested in them
+        if type(value) is str:
```

Comment:
`if isinstance(value, str)` is more intuitive.

Reply:
done